### PR TITLE
chore: standardize ubuntu versions to jammy

### PR DESCRIPTION
**Pull request by matt-boris**
>Updates ubuntu image versions to `jammy`.

### CODE REVIEW

File: workers/docker-worker/test/fixtures/create_images.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function run(name, args) {
 }
 
 async function main() {
-  const IMAGE_NAME = 'ubuntu:18.04';
+  const IMAGE_NAME = 'ubuntu:jammy';
```
Comment (lotas) [inline workers/docker-worker/test/fixtures/create_images.js:56]: aren't we going to drop docker-worker? :)

**Review by lotas — DISMISSED** [review]
Thank you!

File: services/web-server/README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Variables:
     "deadline": "2018-03-07T06:03:06.683Z",
     "expires": "2019-03-07T06:03:06.683Z",
     "payload": {
-      "image": "ubuntu:13.10",
+      "image": "ubuntu:latest",
```
Comment (petemoore) [inline services/web-server/README.md:107]: not ubuntu:jammy?

File: ui/docs/manual/tasks/README.mdx [hunk 3]
```diff
@@ -29,7 +29,7 @@ metadata:
   owner:            nobody@example.com
   source:           https://github.com/taskcluster/taskcluster
 payload:
-  image:            ubuntu:18.04
+  image:            ubuntu:latest
```
Comment (petemoore) [inline ui/docs/manual/tasks/README.mdx:32]: Also here?

File: services/web-server/README.md (hunk 2 above)
Comment (matt-boris) [reply services/web-server/README.md:107]: Nah, these are just examples in the README. Fine to leave as latest.

Comment (matt-boris) [reply services/web-server/README.md:107]: Better to have our examples set to latest instead of v13 lol

**Review by petemoore — DISMISSED** [review]

**Review by lotas — APPROVED** [review]
👏  thanks!

**Pull request merged**